models/tace/test_tace_kappa.py

```
@@ -0,0 +1,169 @@
+model_name = "TACE-v1-OAM-M"
+if False: # If you can autodownload, set this to True
+    try:
+        from tace.foundations import tace_foundations
+        model_path = tace_foundations[model_name]
+    except Exception as e:
+        raise RuntimeError(
+            "Failed to load TACE-v1-OAM-M.\n"
+            "Please manual download the model from:\n"
+            "https://huggingface.co/xvzemin/tace-foundations/resolve/main/TACE-v1-OAM-M.pt\n, "
+            "and put the model into ~/.cache/tace/TACE-v1-OAM-M.pt"
+        ) from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)